src/styles.css

```diff
@@ -138,3 +138,7 @@ html, body {
 .cursor-pointer {
   cursor: pointer;
 }
+
+.red {
+  color: red;
+}
```

Comment:
I think this is still here from when we were testing the ng-simple-file-tree. This should be removed.

Reply:
No it's actually used in the compare-tree component